connectors/webhook/element-templates/webhook-connector-start-event.json

```diff
@@ -261,13 +269,86 @@
       "optional": true,
       "binding": {
         "type": "zeebe:property",
-        "name": "inbound.jwt.requiredPermissions"
+        "name": "inbound.auth.jwt.requiredPermissions"
       },
       "condition": {
         "property": "authorizationType",
         "equals": "JWT"
       }
     },
+    {
+      "label": "Username",
+      "description": "Username for basic authentication",
+      "type": "String",
+      "group": "authorization",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.auth.username"
+      },
+      "condition": {
+        "property": "authorizationType",
+        "equals": "BASIC"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Password",
+      "description": "Password for basic authentication",
+      "type": "String",
+      "group": "authorization",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.auth.password"
+      },
+      "condition": {
+        "property": "authorizationType",
+        "equals": "BASIC"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "API Key",
+      "description": "Expected API key",
+      "type": "String",
+      "group": "authorization",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.auth.apiKey"
+      },
+      "condition": {
+        "property": "authorizationType",
+        "equals": "APIKEY"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "API Key locator",
+      "description": "A FEEL expression that extracts API key from the request",
+      "type": "String",
+      "group": "authorization",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.auth.apiKeyLocator"
+      },
+      "condition": {
+        "property": "authorizationType",
+        "equals": "APIKEY"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "value": "=split(request.headers.authorization, \" \")[2]"
+    },
```

Comment:
What if the API key is in the request body (like Igor mentioned [above](https://github.com/camunda/connectors-bundle/pull/816#issuecomment-1639737000), or, even worse, in a nested data structure) - then FEEL expression is our only solution. I would prefer to keep it as generic as possible for now, this will probably make our lives easier with protocol connectors 🙂

Reply:
@Oleksiivanov We could always provide a service specific protocol conntector template that comes with the FEEL expression as a hidden property right?